docs/welcome.md

```diff
@@ -1,54 +1,59 @@
 
-- [Set up entities and fields](/set-up-entities/)
-- [Configure roles and permissions](/configure-roles-and-permissions/)
-- [Create and manage custom API endpoints and data structures](/custom-types-and-actions/)
-- [Extend capabilities with plugins](/add-plugins-service/)
-- [Commit changes and build](/commit-and-build-new-versions/)
+## Embed Your Organization's Best Practices
 
-## Key Features
+Define and maintain standardized service architectures through live templates and private plugins.
+Create template with pre-configured settings, code generator versions, and plugins.
 
-- **[AI-powered Development](/amplication-ai/)**: Generate production-ready .NET or Node.js code, get intelligent suggestions for your service's architecture, and more with Amplication AI.
-- **[Import Your Existing DB](/how-to/import-prisma-schema/)**: Use your _existing_ databases for a head start on development.
-- **[Full Stack Code Generation](/getting-started/)**: Fully-functional REST and GraphQL APIs, database models, authentication, logging, React Admin UI, and more.
-- **[Event-Driven Architecture](/concepts/event-driven-architecture/)**: Create new services with a modern architecture or modernize your existing apps into a event-driven microservices architecture.
-- **[Seamless Git Integration](/smart-git-sync/)**: Smart git sync with GitHub, Bitbucket, GitLab, or AWS CodeCommit.
-- **[Custom Code Integration](/custom-code/)**: Extend generated code with your own custom business logic.
-- **[Plugin System](/getting-started/plugins/)**: Extend functionality with our community plugins or create your custom plugins.
-- **[Multiple Databases](/getting-started/#databases)**: Support for PostgreSQL, MySQL, MongoDB, MS SQL Server, and more.
-- **Deployment & CI/CD**: Simplify and automate your CI/CD pipeline with our plugins and streamline deployments to popular options like [AWS ECS](/deploy/aws/ecs), [Docker Desktop](/deploy/docker-desktop/), and [Kubernetes](/deploy/kubernetes).
+- **Live Service Templates**
+- **[Private Plugins](/getting-started/plugins/)**: Extend functionality with private plugins or use our community plugins.
+- **Platform Console**
+
+## Technical Debt Prevention
 
-## Tutorials and Samples
+Track and manage technical debt through our Platform Console and Service Catalog.
+Monitor template versions, plugin updates, and dependency changes across all services.
+Receive automatic notifications when updates are available and get clear upgrade paths for outdated services.
+Maintain consistent standards across your entire service portfolio.
 
-- Follow our [step-by-step tutorials](/tutorials/) to build real-world applications.
-- Explore our [sample applications](https://github.com/amplication/sample-app) on GitHub.
+- Technical Debt Management
+- Service Catalog
+- **[Seamless Git Integration](/smart-git-sync/)**: Smart git sync with GitHub, Bitbucket, GitLab, or AWS CodeCommit.
```

Comment:
Not sure this is relevant here (unless rephrase it that all the service updates due to technical debts handling- is available in Git or such